src/event-cache/__tests__/EventCache.test.ts

```diff
@@ -12,7 +12,7 @@ const getSession = jest.fn(() => ({
     eventCount: 1
 }));
 const getUserId = jest.fn(() => 'b');
-const getAttributes = jest.fn(() => {});
+const getAttributes = jest.fn(() => undefined);
```

Comment:
nit: I think the intent with `{}` (here and elsewhere) was to return an empty object.

Clearly it's not being used though... so could even just do
```
const getAttributes = jest.fn();
```
or omit `getAttributes`.

Reply:
`() => {}` does not return an empty object, it returns `undefined`

In order to return an empty object one has to do: ```() => ({})``` (note the extra parenthesis)